filereader/texttemplate/texttemplate.go

```diff
@@ -21,9 +22,27 @@ func Parse(filename string, funcs template.FuncMap) (*template.Template, error)
 			}
 			return content, nil
 		},
+		"toJSON": func(v interface{}) (string, error) {
+			data, err := json.Marshal(v)
+			return string(data), err
+		},
+		"execTemplate": func(name string, ctx interface{}) (string, error) {
```

Comment:
that is a workaround for `template` not being a real function :)  You can not pipe or capture its result